src/components/ec-summary-info/ec-summary-info.spec.js

```diff
@@ -0,0 +1,191 @@
+import { mount } from '@vue/test-utils';
+
+import EcSummaryInfo from './ec-summary-info.vue';
+
+describe('EcSummaryInfo', () => {
+  function mountSummaryInfo(props, mountOpts) {
+    return mount(EcSummaryInfo, {
+      props: {
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  describe('icon', () => {
+    it('should render with the given "icon" prop', () => {
+      const wrapper = mountSummaryInfo({ iconName: 'simple-sell' });
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon').exists()).toBe(true);
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon__simple-sell').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should not render with the not given "icon" prop', () => {
+      const wrapper = mountSummaryInfo();
+      expect(wrapper.findByDataTest('ec-summary-info__main-icon').exists()).toBe(false);
+    });
+  });
+
+  describe('lineItems', () => {
+    it('should render with the given "lineItems" prop', () => {
+      const wrapper = mountSummaryInfo({
+        lineItems: [
+          {
+            stylePreset: 'title',
+            text: 'The title',
+          },
+          {
+            stylePreset: 'description',
+            text: 'Some text',
+            showIcon: true,
```

Comment:
Icon shows if tooltipText is exists, no need this property.

Reply:
ok, corrected, thanks